WPMedia/fusion-news-theme-blocks
# Added line clamp logic css to simple list

## Description
Simple list cut off on Mobile views

## Jira Ticket
[- [TMEDIA-](https://arcpublishing.atlassian.net/browse/TMEDIA-)](https://arcpublishing.atlassian.net/browse/TMEDIA-651)

## Acceptance Criteria
Acceptance Criteria:
- Restrict content to three lines
- Remove height restrictions 

## Test Steps

1. Checkout this branch `git checkout tmedia-651-list-cutoff`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/simple-list-block`
3. Open the published page of `Test - JC - All Blocks`
4. Inspect the text of the first simple list: 
![image](https://user-images.githubusercontent.com/833461/152880977-5880cda9-6799-4395-a7d4-ff3f8cd40852.png)
5. Replace the text in `h3.simple-list-headline-text` With: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.`
6. Enable responsive mode and select `Pixel 5` as thee target device.
7. Ensure on all breakpoints (especially mobile) that text is not being cut off and ellipses are being applied 3 lines or more.


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [x ] Confirmed all the test steps a reviewer will follow above are working.
- [x ] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x ] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [ x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x ] Confirmed this PR has unit test files
  - [ x] Ran `npm run test`, made sure all tests are passing


## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
